src/main/java/org/sonarsource/sonarlint/ls/settings/ServerConnectionSettings.java

```diff
@@ -84,6 +84,10 @@ public boolean isSonarCloudAlias() {
     return List.of(SONARCLOUD_ALIAS).contains(serverUrl);
   }
 
+  public static boolean isSonarCloudAlias(String serverUrl) {
+    return List.of(SONARCLOUD_ALIAS).contains(serverUrl);
```

Comment:
Nitpick: the method above could be rewritten as `return isSonarCloudAlias(this.serverUrl)`